src/core/FileFormatICC.cpp

```diff
@@ -637,6 +638,10 @@ OIIO_ADD_TEST(FileFormatICC, TestFile)
         OIIO_CHECK_EQUAL(1024, iccFile->lut->luts[0].size());
         OIIO_CHECK_EQUAL(1024, iccFile->lut->luts[1].size());
         OIIO_CHECK_EQUAL(1024, iccFile->lut->luts[2].size());
+
```

Comment:
This is a bit of a nit pick, but most of this patch is targeting "spelling" correction/consistency, but this is introducing additional checks to the test, which may be behaviour changing.

Reply:
These have been added with the effort to read .clf files, We are currently simplifying our CLF PR and that's one of the files that could be changed ahead of the CLF PR. If you feel strong about it I can move these changes to a new PR.